apiserver/facades/client/highavailability/highavailability_test.go

```diff
@@ -177,6 +177,20 @@ func (s *clientSuite) TestEnableHANoErrorForNoAddresses(c *gc.C) {
 	c.Assert(err, jc.ErrorIsNil)
 }
 
+// TestEnableHANoErrorVirtualAddresses verifies that virtual addresses doesn't prevent enabling HA
```

Comment:
A test with ipv6 address should be added.

Reply:
Done.